### PR TITLE
Core 1127 lendctaexp funded update

### PR DESCRIPTION
**Pull request by roger-in-kiva**
`LendCtaExp` component updated to show a correct message when loan is already funded

### CODE REVIEW

File: src/components/LoanCards/Buttons/LendCtaExp.vue [hunk 1]
```diff
@@ -88,13 +88,18 @@
...
+		<!-- Refunded, allSharesReserved button -->
 		<kv-ui-button
 			v-if="showNonActionableLoanButton"
 			class="tw-inline-flex tw-flex-1"
 		>
 			{{ ctaButtonText }}
 		</kv-ui-button>
+
+		<!-- Funded / expired -->
+		<div v-if="isFunded" class="tw-w-full tw-text-center tw-rounded tw-bg-tertiary tw-p-2">
```
Comment (dyersituations) [inline src/components/LoanCards/Buttons/LendCtaExp.vue:100]: Background should be the same as the loan callouts, it's darker currently.

File: src/components/LoanCards/KivaClassicBasicLoanCardExp.vue [hunk 2]
```diff
@@ -145,6 +145,7 @@
 				:is-adding="isAdding"
 				@add-to-basket="addToBasket"
 				class="tw-mt-auto"
+				:class="{ 'tw-w-full' : unreservedAmount < 1 }"
```
Comment (dyersituations) [inline src/components/LoanCards/KivaClassicBasicLoanCardExp.vue:148]: While I doubt we'd ever have less than $1, `<= 0` might be more accurate here?

File: src/components/LoanCards/Buttons/LendCtaExp.vue (hunk 1 above)
Comment (roger-in-kiva) [reply src/components/LoanCards/Buttons/LendCtaExp.vue:100]: oh, having double bg https://github.com/kiva/ui/blob/505fad3374a8f83a8ef491b3fc997a76c4f44fe0/src/components/LoanCards/LoanTags/LoanCallouts.vue#L20

Comment (dyersituations) [reply src/components/LoanCards/Buttons/LendCtaExp.vue:100]: Oh yeah, I think we want `background: #f1f1f1;`

File: src/components/LoanCards/Buttons/LendCtaExp.vue [hunk 3]
```diff
@@ -97,7 +97,11 @@
 		</kv-ui-button>
 
 		<!-- Funded / expired -->
-		<div v-if="isFunded" class="tw-w-full tw-text-center tw-rounded tw-bg-tertiary tw-p-2">
+		<div
+			v-if="isFunded"
+			class="tw-w-full tw-text-center tw-roundedx tw-p-2"
```
Comment (dyersituations) [inline src/components/LoanCards/Buttons/LendCtaExp.vue:102]: tw-rounded

**Review by dyersituations — APPROVED** [review]

**Pull request merged**